telegram-wrapper.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+if [ "${XDG_CURRENT_DESKTOP,,}" == "gnome" ] || [ "${XDG_SESSION_DESKTOP,,}" == "gnome" ]; then
```

Comment:
This strict check ignores non-vanilla Gnome sessions, e.g. on Ubuntu these variables are `ubuntu:GNOME` and `ubuntu-wayland` respectively, Manjaro's Gnome seems to change them too.

Reply:
Can you please specify which of this vars (`XDG_CURRENT_DESKTOP` and `XDG_SESSION_DESKTOP`) contains which value on which session?

Reply:
Session | `XDG_CURRENT_DESKTOP` | `XDG_SESSION_DESKTOP`
---|---|---
Ubuntu | `ubuntu:GNOME` | `ubuntu-wayland`
Ubuntu with `communitheme` | `communitheme:ubuntu:GNOME` | `ubuntu-communitheme-snap-wayland`
Manjaro | `GNOME` | `gnome`
Manjaro with Yaru AUR | `Yaru:ubuntu:GNOME` | `Yaru-wayland`

(`communitheme` is Yaru backported to 18.04; for some reason Yaru requires its own session)

Reply:
Thanks. Matching `gnome` substring should work, then.